modin/pandas/test/test_io.py

```diff
@@ -1117,7 +1081,7 @@ def get_internal_df(df):
 
     @pytest.mark.xfail(
         condition="config.getoption('--simulate-cloud').lower() != 'off'",
-        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #2340",
+        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
```

Comment:
why did you change the reference? let's leave it as it was until we figure out more details